### PR TITLE
clarify SUSE CAP and SUSE SCF

### PR DESCRIPTION
**Pull request by cjschroder**
re https://github.com/SUSE/doc-cap/issues/232

### CODE REVIEW

File: xml/cap_depl_about.xml [hunk 1]
```diff
@@ -39,7 +39,45 @@
...
+    All Docker images for the &sle; builds are hosted on registry.suse.com.
+    These are the commercially-supported images. (Community-supported images for 
+    openSUSE are hosted on <link xlink:href="https://hub.docker.com">
+        Docker Hub</link>.) Product manuals on
+    <link xlink:href="https://www.suse.com/documentation/cloud-application-platform-1/">SUSE Doc: SUSE Cloud Application Platform 1</link> refer
+    to the commercially-supported &sle; version.
+</para>
+
+ <para>
+  &scf; is designed to run on any Kubernetes cluster. This guide describes how
```
Comment (btat) [inline xml/cap_depl_about.xml]: Misalignment between tag sets

File: xml/cap_depl_about.xml [hunk 2]
```diff
@@ -39,7 +39,45 @@
...
+
+<para>
+  The &cf; code base provides the basic functionality. &scf; differentiates
+  itself from other &cf; distributions by running in Linux containers managed
+  by &kube;, rather than virtual machines managed with &bosh;, for greater
+  fault tolerance and lower memory use.
+ </para>
+
+<para>
+    All Docker images for the &sle; builds are hosted on registry.suse.com.
```
Comment (btat) [inline xml/cap_depl_about.xml]: `registry.suse.com` could use some formatting love

File: xml/cap_depl_about.xml [hunk 3]
```diff
@@ -39,7 +39,45 @@
...
+   
+ <para>
+&productname; describes the complete software stack, including the operating 
+system, &kube;, and &scf;. 
+  </para>
+   <para>
+&scf; is comprised of the &sle; builds of the UAA (User Account and
+Authentication) server, &scf;, the Stratos Web user interface, and Stratos Metrics (scheduled for the &scf; 1.3 release).
+</para>
+
```
[the remark this answers is not in the document]
Comment (cjschroder) [reply xml/cap_depl_about.xml]: Hmm, that might make a difference.

File: xml/cap_depl_about.xml [hunk 4]
```diff
@@ -39,7 +39,45 @@
...
+<para>
+  The &cf; code base provides the basic functionality. &scf; differentiates
+  itself from other &cf; distributions by running in Linux containers managed
+  by &kube;, rather than virtual machines managed with &bosh;, for greater
+  fault tolerance and lower memory use.
+ </para>
+
+<para>
```
[the remark this answers is not in the document]
Comment (cjschroder) [reply xml/cap_depl_about.xml]: I don't bother much with spacing and alignment--  we can run `daps-xmlformat -vv -i xml/cap_*` in a separate commit occasionally.

File: xml/cap_depl_about.xml (hunk 2 above)
Comment (cjschroder) [reply xml/cap_depl_about.xml]: I added literal tags-- it does not resolve, so I don't think it should be clickable.

File: xml/cap_depl_about.xml (hunk 4 above)
[the remark this answers is not in the document]
Comment (btat) [reply xml/cap_depl_about.xml]: Oh right, forgot about that.

File: xml/cap_depl_about.xml (hunk 2 above)
Comment (btat) [reply xml/cap_depl_about.xml]: Works for me. Since it's a docker registry it doesn't resolve in a browser unless additional tools are used.

**Pull request merged**